packages/salesforcedx-vscode-lwc/src/messages/i18n.ts

```diff
@@ -23,6 +23,8 @@ export const messages = {
     'To run this command, first install the @salesforce/lwc-dev-server plugin. For more info, see [https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev](https://developer.salesforce.com/docs/component-library/documentation/lwc/lwc.get_started_local_dev).',
   force_lightning_lwc_start_addr_in_use:
     'The local development server could not start because the address is already in use. To resolve either: \n1) Stop the local dev server if it is running in another instance.\n or 2) Change the default port [Configuration for Projects (Optional)](https://developer.salesforce.com/docs/component-library/documentation/en/lwc/lwc.get_started_local_dev_setup).\n or 3) Kill the process running on the specified port.',
+  force_lightning_lwc_inactive_scratch_org:
+    'The local development server could not start because the scratch org is not active. To resolve, run "SFDX: Create a Default Scratch Org" to create a new scratch org.',
```

Comment:
@hbc-bindu Please let me know if we need to update this message. Thanks!

Reply:
@xyc , consider rephrasing as:
The local development server couldn’t start as there’s no active scratch org. Run "SFDX: Create a Default Scratch Org" to create a scratch org.

Reply:
Updated, thanks @hbc-bindu